scripts/preprocessing/05a-apply_ica.py

```diff
@@ -86,6 +77,14 @@ def apply_ica(cfg, subject, session):
                    .loc[tsv_data['status'] == 'bad', 'component']
                    .to_list())
 
+    # Load epochs to reject ICA components.
+    msg = f'Input: {fname_epo_in}, Output: {fname_epo_out}'
+    logger.info(gen_log_message(message=msg, step=5, subject=subject,
+                                session=session))
+
+    epochs = mne.read_epochs(fname_epo_in, preload=True)
+    epochs.drop_bad(cfg.ica_reject)
```

Comment:
revert

Reply:
Didn't fully revert but restored the functionality. Should be good now the way it is!